src/metabase/search/appdb/core.clj

```diff
@@ -92,7 +92,7 @@
     (search.ingestion/populate-index! :search.engine/fulltext)))
 
 (defmethod search.engine/reindex! :search.engine/fulltext
-  []
```

Comment:
@piranha 🙄

Reply:
why are there no error from kondo though?

Reply:
Maybe it only works with regular methods?